.circleci/config.yml

```diff
@@ -15,11 +15,25 @@ jobs:
     - run:
         name: Install dependencies
         command: go mod download
+    - run:
+        name: Install linters
+        command: |
+          go get golang.org/x/tools/cmd/goimports
+          go get golang.org/x/lint/golint
```

Comment:
This executions for using as CLI.
go.sum has cache  these so that installed sources, but cannot use as command only `go mod download`